lib/bom.js

```diff
@@ -714,7 +717,30 @@ declare class AudioContext {
   decodeAudioData(arrayBuffer: ArrayBuffer, decodeSuccessCallback: Function, decodeErrorCallback: Function): void;
   decodeAudioData(arrayBuffer: ArrayBuffer): Promise<AudioBuffer>;
   resume(): Promise<void>;
+}
+
+declare class AudioTimestamp {
+  contextTime: number;
+  performanceTime: number;
+}
+
+declare class AudioContext extends BaseAudioContext {
+  baseLatency: number;
+  outputLatency: number;
+  getOutputTimestamp(): AudioTimestamp;
   suspend(): Promise<void>;
+  close(): Promise<void>;
+  createMediaElementSource(myMediaElement: HTMLMediaElement): MediaElementAudioSourceNode;
+  createMediaStreamSource(myMediaStream: MediaStream): MediaStreamAudioSourceNode;
+  createMediaStreamTrackSource(myMediaStreamTrack: MediaStreamTrack): MediaStreamTrackAudioSourceNode;
+  createMediaStreamDestination(): MediaStreamAudioDestinationNode;
+}
+
+declare class OfflineAudioContext extends BaseAudioContext {
+  startRendering(): Promise<AudioBuffer>;
+  suspend(): Promise<void>;
+  length: number;
+  oncomplete: (ev: any) => any;
 }
```

Comment:
`OfflineAudioContext` is defined here
https://webaudio.github.io/web-audio-api/#offlineaudiocontext